### PR TITLE
Add issue and PR templates

### PR DESCRIPTION
**Pull request by bmagyar**
Copying over from ros2_control

### CODE REVIEW

**Review by homalozoa — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Version [e.g. Foxy]
+- Anything that may be unusual about your environment
+
+**Additional context**
+Add any other context about the problem here, especially include any modifications to ros2_control that relate to this issue.
```
Comment (aprotyas) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: Maybe:
```suggestion
Add any other context about the problem here, especially include any modifications to ros2_control_demos that relate to this issue.
```

File: .github/ISSUE_TEMPLATE/good-first-issue.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Overview of your issue here.
+
+## Instructions
+Hi, this is a `good-first-issue` issue. This means we've worked to make it more legible to people who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
+
+We're interested in helping you take the first step, and can answer questions and help you out along the way. Note that we're especially interested in contributions from underrepresented groups!
+
+We know that creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
+
+If you have contributed before, **consider leaving this PR for someone new**, and looking through our general [bug](https://github.com/ros-controls/ros2_control/labels/bug) issues. Thanks!
```
Comment (aprotyas) [inline .github/ISSUE_TEMPLATE/good-first-issue.md:21]:
```suggestion
If you have contributed before, **consider leaving this PR for someone new**, and looking through our general [bug](https://github.com/ros-controls/ros2_control_demos/labels/bug) issues. Thanks!
```

File: .github/ISSUE_TEMPLATE/good-first-issue.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
+
+### 📋 Step by Step
+
+- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
+
+- [ ] 🗄️ **Create a local workspace** for making your changes and testing [following these instructions](https://docs.ros.org/en/galactic/Tutorials/Workspace/Creating-A-Workspace.html), for Step3 use "Download Source Code" section with [these instructions](https://ros-controls.github.io/control.ros.org/getting_started.html#compiling).
+
+- [ ] 🍴 **Fork the repository** using the handy button at the top of the repository page and **clone** it into `~/ws_ros2_control/src/ros-controls/ros2_control`, [here is a guide that you can follow](https://guides.github.com/activities/forking/) (You will have to remove or empty the existing `ros2_control` folder before cloning your own fork)
```
Comment (aprotyas) [inline .github/ISSUE_TEMPLATE/good-first-issue.md:33]:
```suggestion
- [ ] 🍴 **Fork the repository** using the handy button at the top of the repository page and **clone** it into `~/ws_ros2_control/src/ros-controls/ros2_control_demos`, [here is a guide that you can follow](https://guides.github.com/activities/forking/) (You will have to remove or empty the existing `ros2_control_demos` folder before cloning your own fork)
```

File: .github/ISSUE_TEMPLATE/good-first-issue.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+- [ ] **Checkout a new branch** using `git checkout -b <branch_name>`
+
+- [ ] 🤖 **Apply `pre-commit`** auto formatting, by running `pip3 install pre-commit` and running `pre-commit install` in the ros2_control repo.
```
Comment (aprotyas) [inline .github/ISSUE_TEMPLATE/good-first-issue.md:37]:
```suggestion
- [ ] 🤖 **Apply `pre-commit`** auto formatting, by running `pip3 install pre-commit` and running `pre-commit install` in the ros2_control_demos repo.
```

**Review by aprotyas — COMMENTED** [review]
LGTM with some minor suggestions, mostly `:%s/ros2_control/ros2_control_demos/g` :smile_cat:

**Review by jaron-l — DISMISSED** [review]

**Review by christophfroehlich — APPROVED** [review]

**Pull request merged**